Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM yastdevel/cpp
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
+  openslp-devel \
+  yast2 \
+  yast2-ruby-bindings
```

Comment:
so it seems, that this package requires some more packages in docker's image ... Isn't that an issue in the package's dependencies?